Common/src/CConfig.cpp

```diff
@@ -2038,8 +2038,6 @@ void CConfig::SetConfig_Options() {
   addBoolOption("WRT_PERFORMANCE", Wrt_Performance, false);
   /* DESCRIPTION: Output the tape statistics (discrete adjoint)  \ingroup Config*/
   addBoolOption("WRT_AD_STATISTICS", Wrt_AD_Statistics, false);
-  /* DESCRIPTION: Write the mesh quality metrics to the visualization files.  \ingroup Config*/
-  addBoolOption("WRT_MESH_QUALITY", Wrt_MeshQuality, false);
```

Comment:
There is a place in CConfig.cpp to print messages about deprecated options, look for RELAXATION_FACTOR_ADJFLOW, it would be good to explain the disappearance of the option.